### PR TITLE
[STORM-1058] create CLI kill_workers to kill workers on a supervisor node

### PR DESCRIPTION
**Pull request by zhuoliu**
Kill all the workers running on this supervisor. 
Currently there is no direct way to kill all the running workers on a supervisor node from a command line. In addition, if the supervisor dies, the workers will not suicide automatically. 
Also, this CLI can be called by future node health-check script to cleanly shutdown a supervisor on an problematic node.

### CODE REVIEW

**Comment by redsanket** [comment]
LGTM

File: bin/storm.py [hunk 1]
```diff
@@ -348,6 +348,19 @@ def get_errors(*args):
         jvmtype="-client",
         extrajars=[USER_CONF_DIR, os.path.join(STORM_DIR, "bin")])
 
+def kill_workers(*args):
+    """Syntax: [storm kill_workers]
+
+    Kill the workers running on this supervisor. This command should be run
+    on a supervisor node. If the cluster is running in secure node, then user needs
```
Comment (jerrypeng) [inline bin/storm.py]: nit: "secure mode" not "node"

**Comment by jerrypeng** [comment]
other than my nit comment, looks good to me

**Comment by revans2** [comment]
+1 looks good.

**Pull request merged**